woodwork/utils.py

```diff
@@ -650,10 +650,12 @@ def _infer_datetime_format(dates, n=100):
             "%y/%m/%d",
             "%m/%d/%y %H:%M:%S",
             "%y/%m/%d %H:%M:%S",
+            "%y/%m/%d %H:%M:%S%z",
             "%d/%m/%y",
             "%y/%d/%m",
             "%d/%m/%y %H:%M:%S",
             "%y/%d/%m %H:%M:%S",
+            "%y/%d/%m %H:%M:%S%z",
```

Comment:
This doesn't support timezone for all formats with two digit years, we'd want the timezone appended to the end of every iteration of two digit years. I'd recommend adding it as a secondary check if two digit year inference fails